roles/etcd/templates/etcd-docker.service.j2

```diff
@@ -6,7 +6,7 @@ After=docker.service
 [Service]
 User=root
 PermissionsStartOnly=true
-ExecStart={{ docker_bin_dir | default("/usr/bin") }}/docker run --restart=always \
+ExecStart={{ docker_bin_dir | default("/usr/bin") }}/docker run --restart=on-failure:5 \
```

Comment:
Yeah, these changes are largely ignored by systemd and don't seem to have much affect on most deployments. These are being changed for the Ubuntu 14.04 deployment where @abourget was hitting issues. Note that the restart flag was already being passed for etcd, just needed it for kubelet.